Sources/SnapshotTesting/Strategy/View.swift

```diff
@@ -75,17 +75,35 @@ extension Strategy where Snapshottable == UIView, Format == UIImage {
 
   static func image(precision: Float, size: CGSize?) -> Strategy {
     return SimpleStrategy.image(precision: precision).asyncPullback { view in
-      let initialSize = view.frame.size
+      let initialFrame = view.frame                                                                     
       if let size = size { view.frame.size = size }
       guard view.frame.width > 0, view.frame.height > 0 else {
         fatalError("View not renderable to image at size \(view.frame.size)")
       }
+      if let window = UIApplication.shared.keyWindow, window != view {
+        view.frame.origin = CGPoint(x: .max, y: .max)
```

Comment:
When the origin is `.zero` views layout very strangely because of the window some kind of margins/safe-area influence. Not sure if there's a better way of handling this.